CHANGELOG.md

```diff
@@ -1,6 +1,287 @@
+Finn-Ole Carstens, Alexander Debus, Jian Fuh Ong, Marco Garten, Axel Huebl,
+Sophie Rudat (Koßagk), Anton Lebedev, Felix Meyer, Pawel Ordyna, Richard Pausch,
+Franz Pöschel, Adam Simpson, Sebastian Starke, Klaus Steiniger, René Widera
+for contributions to this release!
+
+### Changes to "0.4.0"
+
+**User Input Changes:**
+ - Particle pusher acceleration #2731
+ - stop moving window after N steps #2792
+ - Remove unused ABSORBER_FADE_IN_STEPS from .param files in examples #2942
+ - add namespace "radiation" around code related to radiation plugin #3004
+ - Add a runtime parameter for window move point #3022
+ - Ionization: add silicon to pre-defines #3078
+ - Make dependency between boundElectrons and atomicNumbers more explicit #3076
+ - openPMD: use particle id naming #3165
+ - Docs: update `species.param` #2793 #2795
+
+**New Features:**
+ - PIC:
+   - Particle pusher acceleration #2731
+   - Stop moving window after N steps #2792
+   - Auto domain adjustment #2840
+   - Add a wrapper around main() to catch and report exceptions #2962
+   - Absorber perfectly matched layer PML #2950 #2967
+   - Make dependency between boundElectrons and atomicNumbers more explicit #3076
+ - PMacc:
+   - `ExchangeTypeNames` Verify Parameter for Access #2926
+   - Name directions in species buffer warnings #2925
+   - Add an implementation of exp for pmacc vectors #2956
+   - SimulationFieldHelper: getter method to access cell description #2986
+ - plugins:
+   - PhaseSpaceData: allow multiple iterations #2754
+   - Python MPL Visualizer: plot for several simulations #2762
+   - Emittance Plugin #2588
+   - DataReader: Emittance & PlotMPL: Emittance, SliceEmittance, EnergyWaterfall #2737
+   - Isaac: updated for particle rendering #2940
+   - Resource Monitor Plugin: Warnings #3013
+   - Transition radiation plugin #3003
+   - Add output and python module doc for radiation plugin #3052
+   - Add reference to thesis for emittance plugin doc #3101
+   - Plugins: ADIOS & PhaseSpace Wterminate #2817
+   - Calorimeter Plugin: Document File Suffix #2800
+ - tools:
+   - Support alpaka accelerator `threads` #2701
+   - Add getter for omega and n to python module #2776
+   - Python Tools: Incorporate sim_time into readers and visualizers #2779
+   - Add PIConGPU memory calculator #2806
+   - Python visualizers as jupyter widgets #2691
+   - pic-configure: add `--force/-f` option #2901
+   - Correct target thickness in memory calculator #2873
+   - CMake: Warning in 3.14+ Cache List #3008
+   - Add an option to account for PML in the memory calculator #3029
+   - Update profile hemera-hzdr: CMake version #3059
+   - Travis CI: OSX sed Support #3073
+   - CMake: mark cuda 10.2 as tested #3118
+   - Avoid bash completion file path repetition #3136
+   - Bashcompletion #3069
+   - Jupyter widgets output capture #3149
+   - Docs: Add ionization prediction plot #2870
+   - pic-edit: clean cmake file cache if new param added #2904
+   - CMake: Honor _ROOT Env Hints #2891
```

Comment:
Formatting suggstion: for the next changelog update in the release branch, maybe format as `_ROOT` here (see issue in GitHub preview)